en/08_Changelogs/5.3.0.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 5.3.0 (unreleased)
+---
+
+# 5.3.0 (unreleased)
+
+## Overview
+
+- [Features and enhancements](#features-and-enhancements)
+  - [Validation for elemental content blocks when saving individual blocks](#elemental-validation)
+- [Bug fixes](#bug-fixes)
+
+## Features and enhancements
+
+### Validation for elemental content blocks when saving individual blocks {#elemental-validation}
```

Comment:
```suggestion
### Validation for inline-editable elemental content blocks {#elemental-validation}
```

I doubt we'll release this without resolving the second half of it, which is validation when saving the page. It would be worse to provide sometimes-validation than to keep it with no validation.
Let's assume we'll release both at once.

Reply:
Updated